### PR TITLE
templates: addition of `file_list` macro

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
* Adds a new macro called `file_list` which creates a table showing
  the file list.

Signed-off-by: Javier Delgado <javier.delgado.fernandez@cern.ch>

### CODE REVIEW

**Comment by JavierDelgadoFernandez** [comment]
Addressed in inveniosoftware/invenio-files-rest#8.

**Comment by lnielsen** [comment]
Can you rebase?

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen Rebased.

File: invenio_records_ui/templates/invenio_records_ui/_macros.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    {% endif %}
+    </li>
+  {% endfor %}
+{%- endmacro %}
+
+{%- macro file_list(files) %}
+  <table class="table table-striped">
+    <thead>
+      <tr class="">
+        <th>Name</th>
```
Comment (jirikuncar) [inline invenio_records_ui/templates/invenio_records_ui/_macros.html]: Please wrap it in `{{ _(...) }}`

File: invenio_records_ui/templates/invenio_records_ui/_macros.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <th>Date</th>
+        <th>Size</th>
+        <th></th>
```
Comment (jirikuncar) [inline invenio_records_ui/templates/invenio_records_ui/_macros.html]: `{{ _('Action') }}`

File: tests/test_invenio_views.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    )
+    InvenioRecordsUI(app)
+
+    with app.test_request_context():
+        files = [
+            {
+                'uri': 'http://domain/test1.txt',
+                'name': 'test1.txt',
+                'size': 10,
+                'date': '10 Jul 2016',
```
Comment (jirikuncar) [inline tests/test_invenio_views.py]: are we using ISO 8601?

**Comment by JavierDelgadoFernandez** [comment]
Updated.

**Comment by lnielsen** [comment]
Integrated in Previewer instead.

**Pull request closed without merging**